# Translations template for designate.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the designate project.
#
# Translators:
# Rodrigo Felix de Almeida <rodrigofelixdealmeida@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Designate\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-01-23 06:08+0000\n"
"PO-Revision-Date: 2015-01-23 01:52+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/projects/p/"
"designate/language/pt_BR/)\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: designate/dnsutils.py:98 designate/agent/service.py:92
#, python-format
msgid "Failed to deserialize packet from %(host)s:%(port)d"
msgstr "Falha ao deserializar pacote do %(host)s:%(port)d"

#: designate/dnsutils.py:222 designate/agent/service.py:178
#, python-format
msgid "Unhandled exception while processing request from %(host)s:%(port)d"
msgstr ""
"Exception não-tratada enquanto processando requisição do %(host)s:%(port)d"

#: designate/agent/axfr.py:58
msgid "There was a problem with the AXFR"
msgstr ""

#: designate/api/middleware.py:267
msgid "Missing context in request, please check."
msgstr "Faltando contexto na requisição, por favor verifique."

#: designate/backend/impl_ipa/__init__.py:400
msgid ""
"Error: could not authenticate to IPA - please check for correct keytab file"
msgstr ""
"Erro: não foi possível autenticar no IPA - por favor, verifique para "
"corrigir o arquivo do keytab"

#: designate/backend/impl_ipa/auth.py:55 designate/backend/impl_ipa/auth.py:60
#, python-format
msgid "caught kerberos exception %r"
msgstr "Capturada exceção do kerberos %r"

#: designate/manage/tlds.py:138
#, python-format
msgid "Number of errors: %d"
msgstr "Número de erros: %d"

#: designate/manage/tlds.py:141
#, python-format
msgid ""
"Error Lines:\n"
"%s"
msgstr ""
"Linhas de erros:\n"
"%s"

#: designate/mdns/handler.py:120
#, python-format
msgid "got exception while handling axfr request. Question is %(qr)s"
msgstr "obtida exceção enquanto processando requisição axfr. Questão é %(qr)s"

#: designate/network_api/neutron.py:126
#, python-format
msgid "Failed calling Neutron %(region)s - %(endpoint)s"
msgstr "Falha ao chamar Neutron %(region)s - %(endpoint)s"

#: designate/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "em chamada de duração fixa de loop"

#: designate/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "em chamada de loop dinâmico"

#: designate/openstack/common/policy.py:563
#: designate/openstack/common/policy.py:843
#, python-format
msgid "Failed to understand rule %s"
msgstr "Falha ao entender regra %s"

#: designate/openstack/common/policy.py:573
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Nenhum manipulador para correspondências do tipo %s"

#: designate/openstack/common/service.py:270
msgid "Unhandled exception"
msgstr "Exceção não-tratada"

#: designate/pool_manager/service.py:274
msgid ""
"An unhandled exception in periodic recovery occurred.  This should never "
"happen!"
msgstr ""

#: designate/pool_manager/service.py:297
msgid ""
"An unhandled exception in periodic sync occurred.  This should never happen!"
msgstr ""
